### ISSUE TITLE
Should the WinApps documentation include instructions for setting it up with VirtualBox?

### ISSUE THREAD

**Issue by ManuelFte** [winapps-org/winapps#294]
I managed to set up WinApps with a Windows 11 VM running on VirtualBox, and I was wondering if you'd like me to send you a pull request with the instructions.

I understand that KVM is the preferred method due to being native, standard, and having better performance, but I think VirtualBox would be an interesting option for people like me who are more used to it and its features, or who may even already have a Windows VM set up on it and would rather just connect WinApps to it rather than install a new VM on different software just for this purpose.

I can add a notice at the top advising users to consider the Docker or Virtual Machine Manager methods due to the aforementioned reasons before choosing this one.

**Comment by martinszeltins** [comment]
Yes, please! I think that we are missing a lot of guide material for setting up winapps in general. It would also be nice to see some videos on YouTube about how to do this using VirtualBox, what do you think?

**Comment by oskardotglobal** [comment]
I think the option to use a preexisting VirtualBox VM would be useful, but I don't think we should encourage new users to use it in any way.

**Comment by martinszeltins** [comment]
> I think the option to use a preexisting VirtualBox VM would be useful, but I don't think we should encourage new users to use it in any way.

What is the reason why we shouldn't encourage users to use VirtualBox VM?

**Comment by oskardotglobal** [comment]
Well, VirtualBox is a Type 2 Hypervisor which is not ideal for performance, we want as little of a VM overhead as possible. VirtualBox offers a marginal improvement in simplicity over virt-manager whilst taking a bigger hit in performance.

**Comment by oDelod** [comment]
I'll be honest, I'm used to virtualBox, and I was very interested in the winapps instruction for virtualBox

**Comment by ManuelFte** [comment]
> Yes, please! I think that we are missing a lot of guide material for setting up winapps in general. It would also be nice to see some videos on YouTube about how to do this using VirtualBox, what do you think?

Unfortunately, I don't make videos, but I hope someone is interested in making some.

> Well, VirtualBox is a Type 2 Hypervisor which is not ideal for performance, we want as little of a VM overhead as possible. VirtualBox offers a marginal improvement in simplicity over virt-manager whilst taking a bigger hit in performance.

Without going into much detail to avoid turning this into a VirtualBox vs. Virtual Machine Manager discussion, I installed Windows 11 and configured it with WinApps three times, on Virtual Machine Manager, Docker, and VirtualBox, and I wouldn't say the difference between the virt-manager and VBox installations was only marginal. The virt-manager method was a considerably longer and more complex process than on VBox. I also have decent hardware and run a minified Windows 11, so I didn’t detect any noticeable difference in performance between the two; although my tests weren't exhaustive and my usage of the VM isn't heavy.

I'm sure the virt-manager method is preferable in more scenarios, especially when run on computers with low resources, but I'm also sure that many users will find value in a VBox setup and consider the trade-off between performance and simplicity worth it.

I'll mention all of this in the instructions so that users are aware of the pros and cons beforehand and don’t run into an accidental bad experience.

Since there are many Windows 11 + VirtualBox tutorials on the internet, and the WinApps setup doesn’t require any special configuration during the installation of the OS, with all the configurations being performed afterward, what do you think if I skip all the part about OS installation and simply mention the necessary steps to configure the finished VM for WinApps? This would also align with your vision that it should be more focused on existing VBox users.

I can either link some installation guides for the primary distros or ask users to look for a guide of their preference if you'd rather not link external websites, whichever approach you think is best.

> I'll be honest, I'm used to virtualBox, and I was very interested in the winapps instruction for virtualBox

Great! Then you'll find this guide useful.

**Comment by freechelmi** [comment]
One great advantage for virtualbox is that you can deploy an ova file with preinstalled complete VM that anybody can have running in 3 mouse clics

**Comment by oskardotglobal** [comment]
... which we probably couldn't redistribute

**Comment by freechelmi** [comment]
True , I definitely prefer the quickemu approach

**Comment by zaxlofful** [comment]
This sounds good honestly!

@ManuelFte : If you already know how to do it, a guide that matches the one for [virt-manger](https://github.com/winapps-org/winapps/blob/main/docs/libvirt.md) would probably be helpful.
